test/evergreen.yml

```diff
@@ -2575,17 +2575,18 @@ tasks:
     commands:
       - func: "get project"
       - func: "compile wiredtiger"
+        # CC is set to the system default "clang" binary here as a workaround. Change it back
```

Comment:
Can you add at the beginning of the comment `FIXME-WT-8482` please?

Reply:
Done: a23c9e95e1b28d4afeff3da9100e67db4cf318e0